api/src/main/java/io/opencensus/metrics/Value.java

```diff
@@ -191,6 +206,41 @@ static ValueDistribution create(Distribution value) {
     abstract Distribution getValue();
   }
 
-  // TODO(songya): Add support for Summary type.
-  // This is an aggregation that produces percentiles directly.
+  /**
+   * {@link ValueDistribution} contains summary statistics for a population of values. It optionally
+   * contains a histogram representing the distribution of those values across a set of buckets.
```

Comment:
This should be "It contains a snapshot representing values calculated over an arbitrary time window."

Reply:
done.